litentry/core/assertion-build/src/lib.rs

```diff
@@ -36,23 +36,23 @@ use itc_rest_client::{error::Error as HttpError, RestPath};
 use serde::{Deserialize, Serialize};
 use std::{fmt::Debug, str, string::String};
 
-use itp_types::AccountId;
+pub mod assertion1;
+pub mod assertion2;
 
-use litentry_primitives::{Identity, Ruleset};
-
-pub mod discord;
-
-use crate::discord::ruleset2_verification;
+pub use crate::{assertion1::build_assertion1, assertion2::build_assertion2};
```

Comment:
My personal preference would be something like `a1::build`, and we only export the mod name but not the fn name.

So from outside it's `lc_assertion_build::a1::build()`, I think that's less verbose while still accurate